### ISSUE TITLE
modal does not close when OK pressed (in IE 11)

### ISSUE THREAD

**Issue by Youdaman** [chenz24/vue-blu#38]
IE 11 console says `SCRIPT438: Object doesn't support property or method 'remove'`

**Comment by chenz24** [comment]
Yes, there are some problem in compatible with ie11, I will fix it. Recently I'm busy in doing other things so maybe it will take a few days longer.
